README.md

```diff
@@ -1 +1,170 @@
+
+## 🛠 Built With <a name="built-with"></a>
+Using React documentation
+Using React components
+Using React props
+Using React Router
+Connect React and Redux
+Handle events in a React app
+Write unit tests with React Testing Library
+Use styles in a React app
+Use React hooks
+Use store, actions and reducers in React
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  React
+  Redux
+  CSS  
+  </details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Display a list of stocks according to filter. 
+- Stock history with company details and website link.
+- Real-Time Stock-date: Access up-to-date information directly from the Financial modeling API.
+- User-Friendly Interface: An intuitive and visually appealing interface for an exceptional user experience.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project, you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/George7h/investgrow-hub
+
+
+### Install
+
+Install this project with: 
+- Git
+- Node.js
+- Node package manager
+- React script
+
+
+install the projects depandancies:
+> npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+> npm start
+
+### Run tests
+
+> npm test
+
+### Deployment
+
+Project has been deployed by using the Render however it can be deployed by gh-pages and other web platforms.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **George Hamman**
+
+- GitHub: [@githubhandle](https://github.com/George7h)
+- LinkedIn: <a href="https://www.linkedin.com/in/george-hamman-95b98224b/">George Hamman</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add stock bought at date and calculate the earnings if that stock was invested at that date.
+- Add stock to watchlist
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/George7h/investgrow-hub/issues/new).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give a ⭐️  
+
+<p align="right">(<a href="#readme-top">back to top </a>)</p>
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my code reviewers & Microverse management
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] As stated in the project requirement kindly make sure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).